tests/TestHelpers/WebDavHelper.php

```diff
@@ -747,7 +747,7 @@ public static function getDavPath(
 			if ($type === "trash-bin") {
 				return "/remote.php/dav/spaces/trash-bin/" . $spaceId . '/';
 			}
-			return "dav/spaces/" . $spaceId . '/';
+			return "/remote.php/dav/spaces/" . $spaceId . '/';
```

Comment:
@nabim777 link to the issue?

Reply:
https://github.com/owncloud/ocis/issues/6954

Reply:
https://github.com/owncloud/ocis/issues/6954#issuecomment-1663766325